models/object_detection_nanodet/NanodetPlus.py

```diff
@@ -0,0 +1,138 @@
+import cv2
+import numpy as np
+
+class NanoDet():
+    def __init__(self, modelPath, prob_threshold=0.35, iou_threshold=0.6):
+        with open('coco.names', 'rt') as f:
+            self.classes = f.read().rstrip('\n').split('\n')
```

Comment:
Lets keep all classes here in this file like [this](https://github.com/open-mmlab/mmdetection/blob/df28da98926bc410e16bed1e9fc7d425d9a89495/mmdet/datasets/coco.py#L25-L38)

Reply:
- Resolved the changes and replaced the file with classes in the demo.py file
- Removed file 'coco.names'